dispatch/static/manager/src/js/components/ContentEditor/embeds/VideoEmbed.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+
+import { FormInput, TextInput } from '../../inputs'
+
+class VideoEmbedComponent extends React.Component {
+
+  constructor(props){
+    super(props)
+
+    this.handleUrlChange = this.handleUrlChange.bind(this)
+    this.insertVideo = this.insertVideo.bind(this)
+
+    this.state = {
+      inserted: false,
+      validURL: false
+    }
+  }
+
+  handleUrlChange(e){
+    e.preventDefault()
+    this.props.updateField('url', e.target.value)
+    console.log("URL IS: "+this.props.data.url);
```

Comment:
I hadn't yet done final changes when the PR was made. Will remove all the logs etc.